vault/data_source_aws_static_credentials_test.go

```diff
@@ -0,0 +1,56 @@
+package vault
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-provider-vault/internal/consts"
+	"github.com/hashicorp/terraform-provider-vault/internal/provider"
+	"testing"
+)
+
+func TestAccDataSourceAWSStaticCredentials(t *testing.T) {
+	//mountPath := acctest.RandomWithPrefix("tf-test-aws-static")
+	//accessKey, secretKey := testutil.GetTestAWSCreds(t) // from the environment, causes test to skip if unset
+
+	resource.Test(t, resource.TestCase{
+		IsUnitTest: false,
```

Comment:
We generally add in our acceptance test check in the PreChecks below, similar to what is done [here](https://github.com/hashicorp/terraform-provider-vault/blob/main/vault/data_source_ldap_static_role_credentials_test.go#L26)

Doesn't mean this won't work or is wrong, just wanted to call out the pattern in the codebase :)